pandas/core/frame.py

```diff
@@ -8648,12 +8653,15 @@ def blk_func(values):
 
             # After possibly _get_data and transposing, we are now in the
             #  simple case where we can use BlockManager.reduce
-            res = df._mgr.reduce(blk_func)
-            out = df._constructor(res,).iloc[0].rename(None)
+            res, indexer = df._mgr.reduce(blk_func, ignore_failures=ignore_failures)
+            out = df._constructor(res).iloc[0]
```

Comment:
we don't need to rename?

Reply:
changed the Index in BlockManager.reduce from `Index([0])` to `Index([None])` to make the rename unnecessary